libs/stokesian_dynamics/sd.hpp

```diff
@@ -37,20 +37,15 @@ typedef r123::Philox2x64 RNG;
  *  thermalizer functor.
  *
  *  To understand the Stokesian Dynamics method, it is recommended to read
- *  Durlofski et al. (1987), because it describes the method that is
+ *  @cite durlofsky87a, because it describes the method that is
  *  implemented here, apart from the thermalization.
- *      "Dynamic simulation of hydrodynamically interacting suspensions"
- *      https://core.ac.uk/download/pdf/4895234.pdf
  *
- *  All references to formulae in this file refer to the aforementioned paper,
- *  unless otherwise noted.
+ *  All references to formulae in this file refer to the paper 
+ *  @cite durlofsky87a, unless otherwise noted.
  *
  *  As they state in the paragraph below equation (2.17) it may be useful to
- *  also read Jeffrey et al. (1984) to better understand the notation used to
+ *  also read @cite jeffrey84a to better understand the notation used to
  *  compute the grand mobility matrix in this Stokesian Dynamics method.
- *      "Calculation of the resistance and mobility functions for two unequal
- *      rigid spheres in low-Reynolds-number flow"
- *      https://doi.org/10.1017/S0022112084000355
```

Comment:
Please don't remove these references.  You can *add* BibTeX on top, but SD should remain independent of ESPResSo and that includes the in-code documentation.

Reply:
Fair point, I'll add them back into the code